tests/regression/lp-1898038/test-docker-support-app/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: test-docker-support-app
```

Comment:
We usually call such snaps `test-snapd-...` as that pattern is filtered out in various places not to skew stats.

Reply:
Thanks, fixed in 1deaf49cd7